acceptance/ui/features/steps/pools_steps.rb

```diff
@@ -211,28 +220,23 @@ def addPoolJson(pool)
     addPool("table://pools/" + pool + "/name", "table://pools/" + pool + "/description")
 end
 
-def removeAllPools()
-    removeAllPoolsCLI()
-end
-
 def removeAllPoolsExceptDefault()
     visitApplicationsPage()
     removeAllEntries("service")
     removeAllHostsCLI()
     removeAllPoolsCLI()
-    addDefaultPool()
     refreshPage()
 end
 
 def removeAllPoolsCLI()
```

Comment:
Can we rename this to 'removeAllPoolsExceptDefaultCLI' now? Because now it does not remove all the pools...